docs/development/variants.md

```diff
@@ -0,0 +1,51 @@
+[👈 Back to README](../../README.md)
+
+# Variants
+
+Test Pilot has the ability to run variant tests on both the Test Pilot
+site and for individual experiments. Information about testing experiments
+is [here](../experiments/variants.md).
+
+To add a variant test to Test Pilot itself, edit the
+`frontend/app/src/reducers/varianttests.js` file. Add another object to the
```

Comment:
`frontend/src/app/reducers/varianttests.js`? (bonus points if we can link to the file)